package.json

```diff
@@ -46,7 +46,7 @@
     "unit-haproxy": "cd haproxy/tests && make test",
     "wdio-cht-form": "wdio run ./tests/e2e/cht-form/wdio.conf.js",
     "wdio-default-mobile-local": "export VERSION=$(node ./scripts/build/get-version.js) && ./scripts/build/build-service-images.sh && wdio run ./tests/e2e/default-mobile/wdio.conf.js --suite=all",
-    "wdio-local": "export VERSION=$(node ./scripts/build/get-version.js) && ./scripts/build/build-service-images.sh && wdio run ./tests/e2e/default/wdio.conf.js",
+    "wdio-local": "wdio run ./tests/e2e/default/wdio.conf.js",
```

Comment:
Did you mean to commit this change?  (Sorry, I know I may have preemptively jumped on this PR before you were done....)